var/spack/repos/builtin/packages/py-pdoc3/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPdoc3(PythonPackage):
+    """Auto-generate API documentation for Python projects."""
+
+    homepage = "https://pdoc3.github.io/pdoc/"
+    pypi     = "pdoc3/pdoc3-0.10.0.tar.gz"
+
+    # FIXME: Add a list of GitHub accounts to
+    # notify when the package is updated.
+    # maintainers = ['github_user1', 'github_user2']
+
+    version('0.10.0', sha256='5f22e7bcb969006738e1aa4219c75a32f34c2d62d46dc9d2fb2d3e0b0287e4b7')
+
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.10.0:')
```

Comment:
Same here, no need to bother with `when='...'` if you're not planning on adding older releases. You're welcome to keep it though, up to you.